opentelemetry-sdk/CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 ## vNext
 
-- *Breaking* SimpleLogProcessor modified to be generic over `LogExporter` to
-  avoid dynamic dispatch to invoke exporter. If you were using
-  `with_simple_exporter` to add `LogExporter` with SimpleLogProcessor, this is a
-  transparent change.
-  [#2338](https://github.com/open-telemetry/opentelemetry-rust/pull/2338)
+- *Breaking* 
+  - SimpleLogProcessor modified to be generic over `LogExporter` to
+    avoid dynamic dispatch to invoke exporter. If you were using
+    `with_simple_exporter` to add `LogExporter` with SimpleLogProcessor, this is a
+    transparent change.
+    [#2338](https://github.com/open-telemetry/opentelemetry-rust/pull/2338)
+  - `opentelemetry::global::shutdown_tracer_provider()` Removed from the API, should now use `tracer_provider.shutdown()` see [#2369](https://github.com/open-telemetry/opentelemetry-rust/pull/2369) for a migration example.
```

Comment:
can you add something to the tune of
"Tracer provider" is cheaply clone able, so users are encouraged to set a clone of it to the global, so that instrumentations and other components can obtain tracers from global::tracer () ...
The tracer_provider must be kept around to call shutdown on it at the end of application..

Reply:
updated